CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
```

Comment:
Am I doing the right thing here? Or should I leave something specific in place that `cargo-release` requires?

Reply:
No that is exactly what I use for it :+1: (though at some point I should probably adjust the regex to be less whitespace sensitive :D